homeassistant/components/unifi/unifi_entity_base.py

```diff
@@ -33,6 +34,29 @@ async def async_will_remove_from_hass(self) -> None:
         """Disconnect object when removed."""
         self.controller.entities[self.platform.domain][self.TYPE].remove(self.mac)
 
+    async def async_remove(self):
+        """Clean up when removing entity.
+
+        Remove entity if no entry in entity registry exist.
+        Remove entity registry entry if no entry in device registry exist.
+        Remove device registry entry if there is only one linked entity.
```

Comment:
Maybe clarify that this entity is the last one entity in this case?

Reply:
Sure